docker-compose.yaml

```
@@ -0,0 +1,37 @@
+services:
+  backend:
```

Adding this `---` or leaving an empty first line will let you have a `Run All Services` button show up on VSCode making it easy to run compose if you have the official docker extension.

```suggestion
---
services:
  backend:
```